.github/workflows/test-external.yml

```diff
@@ -9,7 +9,7 @@ on:
 
 jobs:
   test:
-    runs-on: macos-latest
+    runs-on: ubuntu-latest
```

Comment:
any reason to switch to ubuntu?

oh, when i checked last time, macos-latest provided more cpus/memory, but it's no longer the case [now](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories):
- ubuntu-latest: 4 cores / 16 gb
- macos-latest: 3 (M1) cores / 7 gb

Reply:
if ubuntu-latest is better, should we switch the test-long task to ubuntu as well?

Reply:
yes might be worth it, for me the driver was faster yices install but more memory and cores is nice too. Although I think eventually we probably want to migrate everything to the Docker workflow